latest/ug/nodes/hybrid-nodes-cluster-update.adoc

```diff
@@ -69,15 +72,15 @@ You can enable EKS Hybrid Nodes in an existing cluster by using:
           - Cidrs: [RemotePodCIDR]
 ----
 
-. Continue with the next step.
+. Continue with <<Next steps: Cluster setup, Next steps>>.
 
 
 [#hybrid-nodes-cluster-enable-cli]
 === Enable EKS Hybrid Nodes in an existing cluster - {aws} CLI
 
-. Run the following command to enable EKS Hybrid Nodes and update an EKS cluster. Before running the command, replace the following with your desired settings. For a full list of settings, see the link:eks/latest/APIReference/API_UpdateClusterConfig.html[UpdateClusterConfig,type="documentation"] in the _Amazon EKS API Reference_.
-.. `CLUSTER_NAME`: name of the EKS cluster to be created.
-.. `AWS_REGION`: {aws} Region where the cluster will be created. 
+. Run the following command to enable EKS Hybrid Nodes by updating an EKS cluster. Before running the command, replace the following with your desired settings. For a full list of settings, see the link:eks/latest/APIReference/API_UpdateClusterConfig.html[UpdateClusterConfig,type="documentation"] in the _Amazon EKS API Reference_.
```

Comment:
Nit: "Run the following command to enable the `RemoteNetworkConfig` for EKS Hybrid Nodes for your EKS cluster"